controllers/validation/accountpool_validation_controller.go

```diff
@@ -0,0 +1,201 @@
+package validation
+
+import (
+	"context"
+	"fmt"
+	awsv1alpha1 "github.com/openshift/aws-account-operator/api/v1alpha1"
+	"github.com/openshift/aws-account-operator/pkg/awsclient"
+	"gopkg.in/yaml.v2"
+	"reflect"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"strconv"
+
+	"github.com/go-logr/logr"
+	"github.com/openshift/aws-account-operator/pkg/utils"
+	"github.com/openshift/aws-account-operator/test/fixtures"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+var logs = logf.Log.WithName("controller_accountpoolvalidation")
+
+const (
+	ControllerName = "accountpoolvalidation"
+)
```

Comment:
Had to change the name of these two variables in order to redeclare them for the accountpool_validation_controller.